pages/_app.tsx

```diff
@@ -12,6 +12,11 @@ export const textFont = localFont({
   ],
 });
 
+export const headingFont = Roboto({
+  weight: "400",
+  subsets: ["latin"],
```

Comment:
Nice one. Do you know what is this option now after implementation?

Reply:
The subsets option is to define the array of preload subset